basic_suite_test.go

```diff
@@ -37,10 +37,52 @@ type BasicTestsSuite struct {
 
 func (s *BasicTestsSuite) SetupSuite() {
 	writer := logrus.StandardLogger().Writer()
+
 	s.options = []*exechelper.Option{
 		exechelper.WithStderr(writer),
 		exechelper.WithStdout(writer),
 	}
+
+	s.Require().NoError(exechelper.Run("kubectl apply -f ./deployments/spire/spire-namespace.yaml", s.options...))
+	s.Require().NoError(exechelper.Run("kubectl apply -f ./deployments/spire", s.options...))
+
+	s.Require().NoError(exechelper.Run("kubectl wait -n spire --timeout=60s --for=condition=ready pod -l app=spire-agent", s.options...))
+	s.Require().NoError(exechelper.Run("kubectl wait -n spire --timeout=60s --for=condition=ready pod -l app=spire-server", s.options...))
+
+	s.Require().NoError(exechelper.Run(`kubectl exec -n spire spire-server-0 --
+												/opt/spire/bin/spire-server entry create
+												-spiffeID spiffe://example.org/ns/spire/sa/spire-agent
+												-selector k8s_sat:cluster:nsm-cluster
+												-selector k8s_sat:agent_ns:spire
+												-selector k8s_sat:agent_sa:spire-agent
+												-node`, s.options...))
+
+	s.Require().NoError(exechelper.Run(`kubectl exec -n spire spire-server-0 --
+												/opt/spire/bin/spire-server entry create
+												-spiffeID spiffe://example.org/ns/default/sa/default
+												-parentID spiffe://example.org/ns/spire/sa/spire-agent
+												-selector k8s:ns:default \
+												-selector k8s:sa:default`, s.options...))
+}
+
+func (s *BasicTestsSuite) TearDownSuite() {
+	s.Require().NoError(exechelper.Run("kubectl delete -f ./deployments/spire/spire-namespace.yaml"))
+}
+
+func (s *BasicTestsSuite) TearDownTest() {
+	s.Require().NoError(exechelper.Run("kubectl delete serviceaccounts --all"))
```

Comment:
What about having exechelper.RunScript(multileneCommand)?

Reply:
Looks like we can do this, because of  shlex lib that is used by exechelper is using newline characters in parsing command arguments https://github.com/google/shlex/blob/master/shlex.go#L79